en/editor/publish/publish-bytedance-mini-game.md

```diff
@@ -0,0 +1,62 @@
+# Publish to ByteDance Mini Games
+
+ByteDance mini-game is developed based on the ByteDance full products, which does not require users to download, and is a brand-new game type that can be played on tap.
+
+The game submission, review and release process of the mini-games need to comply with the requirements and standard processes of the Byte official team. For specific information, please refer to the link at the end of the article.
+
+## Publish ByteDance with Cocos Creator 3D
+
+1. Download **ByteDance DevTools** on [ByteDance Official Document](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download).
+
+2. Log in to [Developer Platform](https://microapp.bytedance.com/)to find appid. For details, please refer to the official details [Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/set-up-mini-game)。
+
+    ![](https://sf1-ttcdn-tos.pstatp.com/obj/website-img/f296a9f80eaeb40f4af38e8a4e25e17e_12.png)
+
+3. Select the **ByteDance Mini Game** in the **Platform** of the **Build** panel, fill in the `mini game appid`, and then click **Build**
+
+    ![](./publish-bytedance-mini-game/build.jpg)
+
+4. Preview game
+
+    Following this process, a `bytedance-mini-game` folder will be generated in the project's **build** directory (the name of the folder is based on the **Build Task Name**),then you can open your d
```

Comment:
,then you can open your d

?? I don't understand this.

Reply:
it's a mistake, I will change it.